ArcGISRuntimeSDKQt_QMLSamples/Routing/FindClosestFacilityToMultipleIncidentsService/FindClosestFacilityToMultipleIncidentsService.qml

```diff
@@ -0,0 +1,238 @@
+                    }
+                }
+
+                Button {
+                    id: resetButton
+                    text: qsTr("Reset")
+                    Layout.margins: 2
+                    Layout.fillWidth: true
+                    enabled: false
+                    onClicked: {
+                        // clear graphics overlay
+                        resultsOverlay.graphics.clear();
+                        // enable solve button
+                        solveButton.enabled = true;
+                        // disable reset button
+                        resetButton.enabled = false;
+                    }
+                }
+            }
+        }
+    }
+
+    // declare picture marker symbol for the facilities feature layer
+    PictureMarkerSymbol {
+        id: facilitySymbol
+        url: "https://static.arcgis.com/images/Symbols/SafetyHealth/FireStation.png"
+        height: 30
+        width: 30
+    }
+
+    // declare picture marker symbol for the incidents feature layer
+    PictureMarkerSymbol {
+        id: incidentSymbol
+        url: "https://static.arcgis.com/images/Symbols/SafetyHealth/esriCrimeMarker_56_Gradient.png"
+        height: 30
+        width: 30
+    }
+
+    // declare simple line symbol for the route graphics
+    SimpleLineSymbol {
+        id: routeSymbol
+        style: Enums.SimpleLineSymbolStyleSolid
+        color: "blue"
+        width: 2.0
+    }
+
+    // declare closest facility task
+    ClosestFacilityTask {
+        id: task
+        url: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ClosestFacility"
+
+        // once loaded created default parameters
+        onLoadStatusChanged: {
+            if (loadStatus !== Enums.LoadStatusLoaded)
+                return;
+
+            task.createDefaultParameters();
+        }
+
+        onCreateDefaultParametersStatusChanged: {
+            if (createDefaultParametersStatus !== Enums.TaskStatusCompleted)
+                return;
+
+            // disable busy indicator
+            busy = false;
+            // enable solve button
+            solveButton.enabled = true;
+            // set facilityParams to closest facilities parmeters
+            facilityParams = createDefaultParametersResult;
+            // set the facilities parameter with the facilitites feature table
+            facilityParams.setFacilitiesWithFeatureTable(facilitiesFeatureTable, params);
+            // set the incidents parameter with the incidents feature table
+            facilityParams.setIncidentsWithFeatureTable(incidentsFeatureTable, params);
+        }
+
+        onSolveClosestFacilityStatusChanged: {
+            if (solveClosestFacilityStatus !== Enums.TaskStatusCompleted)
+                return;
+
+            // loop through all incidnets
+            for (var incidentIndex = 0; incidentIndex < incidentsFeatureTable.numberOfFeaturesAsInt; incidentIndex++) {
```

Comment:
optional: prefer `let` to `var` in for loop declarations. It's one of the newer language features and a good fit for cases like this.

No need to refactor any existing code.